src/features/swap/SwapForm.tsx

```diff
@@ -31,12 +29,20 @@ const initialValues: SwapFormValues = {
 
 export function SwapFormCard() {
   return (
-    <FloatingBox width="w-96" classes="overflow-visible">
-      <div className="flex justify-between mb-5">
-        <h2 className="pl-1 text-lg font-medium">Swap</h2>
+    <FloatingBox
+      width="max-w-md w-full"
+      padding="p-0"
+      classes="overflow-visible border border-primary-dark dark:border-[#333336] dark:bg-[#1D1D20]"
+    >
+      <div className="flex justify-between border-b border-primary-dark dark:border-[#333336] p-6">
+        <h2 className=" text-[32px] leading-10 font-fg font-medium text-primary-dark dark:text-clean-white ">
```

Comment:
Spacing before and after classname. Seems like there's a few of these, there's probably a VSCode plugin that can help :)